x-pack/plugins/monitoring/server/lib/alerts/fetch_missing_monitoring_data.ts

```diff
@@ -73,6 +74,7 @@ export async function fetchMissingMonitoringData(
                 cluster_uuid: clusters.map((cluster) => cluster.clusterUuid),
               },
             },
+            createDatasetFilter('node_stats', 'elasticsearch.node_stats'),
```

Comment:
Since we are querying for `metrics-elasticsearch.node_stats` as part of the index pattern added here https://github.com/elastic/kibana/pull/119112/files#diff-666dd698bc0e64e0a8265f7f08b758b5d73921ea321e0312f32474e3f834ac9bR60, the term filter should have been added.  I presume adding the `node_stats` to begin with is correct and tested that the alert is still working.